internal/charmstore/common_test.go

```diff
@@ -20,11 +21,12 @@ type commonSuite struct {
 // addRequiredCharms adds any charms required by the given
 // bundle that are not already in the store.
 func (s *commonSuite) addRequiredCharms(c *gc.C, bundle charm.Bundle) {
+	c.Logf("addRequiredCharms")
```

Comment:
I'm not sure we still want this message since we worked out what was going wrong.